WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -11489,7 +11489,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = PZYM8XX95Q;
+				DEVELOPMENT_TEAM = S36US5P4XF;
```

Comment:
Because this file isn't part of the changes, can we please remove all changes in it from the PR?